### ISSUE TITLE
Feature - Custom colours for diwako_dui_special_track objects and other non-team colours

### ISSUE THREAD

**Issue by Drofseh** [diwako/diwako_dui#39]
Right now diwako_dui_special_track icons for diwako_dui_special_track objects are bright pink, it would be nice if that colour (and other non-team colours) could be a setting.

**Comment by Drofseh** [comment]
By non-team I mean colours that aren't the RGBY of the squad's team colours.

**Comment by diwako** [comment]
I would like to have the color defined with all other colors in the config, reason behind it, pretty much just coherence between future features.

But I can understand that you'd like to choose for your own, there is a CBA color picker, tho that would mean there is a need for RGB to Hex conversion. Right now I have hex to RGB. IIRC ace had some conversion for it, maybe can ask in their slack to """borrow""" their code.
Not sure right now.

To summarize:
Either predefined color, meaning you can suggest them, or color picker but needs a bit more code.

**Comment by Drofseh** [comment]
Colour picker would be ideal imo.

**Comment by diwako** [comment]
Color picker added in commit ffeae24867aaa0c6fcc51ba95c571ce908566cae
